resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -33,7 +33,7 @@ Item
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         horizontalAlignment: Text.AlignHCenter
-        text: catalog.i18nc("@label", "Ultimaker Account")
+        text: catalog.i18nc("@label", "Sign in to Ultimaker Cloud")
```

Comment:
We cannot use "Ultimaker Cloud" anymore as product name. Should be either "Ultimaker Account" or "Ultimaker Digital Factory".

Reply:
Thanks Chris. Fixed.